tests/spread/commands/init-flask-framework/task.yaml

```diff
@@ -0,0 +1,23 @@
+summary: test charmcraft init with flask-framework profile
+
+execute: |
+  mkdir -p test-init
+  cd test-init
+  charmcraft init --profile flask-framework
+  export CHARMCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS=true
```

Comment:
This can be set in an `environment` keyword for the task. [Here's an example](https://github.com/canonical/charmcraft/blob/main/tests/spread/smoketests/basic-centos-7/task.yaml#L8-L10)

Reply:
Updated, thanks!